### ISSUE TITLE
Doctors need to know the list of patients for whom follow-up has been scheduled. 

### ISSUE THREAD

**Issue by raunaqp** [Intelehealth/intelehealth-doctor-webapp#40]
A new card has to be added. The work flow and details are mentioned below. 

1. Fields on the dashboard: 
Patient ID, Name, Gender, DOB, Location, Follow-up date, Status (patient has come for follow-up visit or they haven't)
2. Once the visit is complete, patients with scheduled follow-up in descending order (most upcoming dates shown first) should be visible here on the UI.  
3. Once the patient has come for the follow-up, it moves to queued visits again, and once the visit is complete, it is removed from the dashboard. 
4. The user should be able to see and filter these follow-up visits on a weekly/ daily basis on the UI. 1. Fields on the dashboard: 
Patient ID, Name, Gender, DOB, Location, Follow-up date, Status (patient has come for follow-up visit or they haven't)
2. Once the visit is complete, patients with scheduled follow-up in descending order (most upcoming dates shown first) should be visible here on the UI.  
3. Once the patient has come for the follow-up, it moves to queued visits again, and once the visit is complete, it is removed from the dashboard. 
4. The user should be able to see and filter these follow-up visits on a weekly/ daily basis on the UI.
